### PR TITLE
Update install guides

### PR DESCRIPTION
**Pull request by lfbrock**
Updating install guides to include a step to check SSL certificate is set up correctly

### CODE REVIEW

**Comment by jasonblais** [comment]
+1

File: source/install/ee-prod-rhel-6.rst [hunk 1]
```diff
@@ -317,6 +317,9 @@ Set up NGINX with SSL (Recommended)
 7. Add the following line to cron so the cert will renew every month
   * ``crontab -e``
   * ``@monthly /home/YOURUSERNAME/letsencrypt/letsencrypt-auto certonly --reinstall -d yourdomainname && sudo service nginx reload``
+8. Check that your SSL certificate is set up correctly
+  * Test the SSL certificate by visiting a site such as `https://www.ssllabs.com/ssltest/index.html <https://www.ssllabs.com/ssltest/index.html>`_
+  * If there’s an error about the missing chain or certificate path, there is likely an intermediate certificate missing that needs to be included
```
Comment (it33) [inline source/install/ee-prod-rhel-6.rst]: People really struggle with this, propose asking CH for review/verification, I think he mentioned [there was a post in forums](https://forum.mattermost.org/t/android-mattermost-client-version-incompatible/2151/4?u=it33) that had the precise instructions to fix.

Comment (lfbrock) [inline source/install/ee-prod-rhel-6.rst]: CH has reviewed, discussed in Leads + current text is okay

**Comment by esethna** [comment]
+1

**Pull request merged**